package.json

```diff
@@ -13,8 +13,9 @@
   },
   "license": "BSD-2-Clause",
   "dependencies": {
+    "nan": "^2.4.0",
     "node-pre-gyp": "^0.6.28",
-    "nan": "^2.4.0"
```

Comment:
Oh, I think yarn automatically sorts packages in alphabetical order when inserting new dependencies.